tests/cache/models/samples/put.js

```diff
@@ -87,7 +87,7 @@ describe(`tests/cache/models/samples/put.js, api: cache: PUT ${path}`, () => {
 
           const _err = res.body.errors[ZERO];
           expect(_err.type).to.equal('ResourceNotFoundError');
-          expect(_err.description).to.equal('Aspect not found.');
+          expect(_err.description).to.equal('Sample not found.');
```

Comment:
Is this the same exact error and message as when using db?

Reply:
actually db says `Aspect not found`
Will double check why the cache says otherwise

Reply:
The message changes from missing Aspect to missing Sample, because in cache the !sample test which throws 404 is performed BEFORE the !aspect test.
Since the redisOps.deleteKeys now actually deletes the samples, the !samples fails before the !aspects check, hence the change in the error code.

Reply:
They should return the same error whether from db or cache